.github/dependabot.yml

```diff
@@ -0,0 +1,71 @@
+version: 2
+updates:
+  # === Root package.json (/) ===
+  # Internal dependencies
+  - package-ecosystem: 'npm'
+    directory: '/'
+    schedule:
+      interval: 'daily'
+    allow:
+      - dependency-name: '@arbitrum/*'
+      - dependency-name: '@offchainlabs/*'
+    versioning-strategy: increase-if-necessary
+    ignore:
+      - dependency-name: '*'
+        update-types: ['version-update:semver-patch']
+    reviewers:
+      - 'OffchainLabs/full-stack'
+    labels:
+      - 'dependencies'
+      - 'internal'
+
+  # External dependencies
+  - package-ecosystem: 'npm'
+    directory: '/'
+    schedule:
+      interval: 'daily'
+    ignore:
+      - dependency-name: '@arbitrum/*'
+      - dependency-name: '@offchainlabs/*'
+    # Disable version updates for external dependencies - security only
+    open-pull-requests-limit: 0
+    reviewers:
+      - 'OffchainLabs/full-stack'
+    labels:
+      - 'dependencies'
+      - 'external'
+
+  # === src/package.json (/src) ===
+  # Internal dependencies
+  - package-ecosystem: 'npm'
+    directory: '/src'
+    schedule:
+      interval: 'daily'
+    allow:
+      - dependency-name: '@arbitrum/*'
+      - dependency-name: '@offchainlabs/*'
+    versioning-strategy: increase-if-necessary
+    ignore:
+      - dependency-name: '*'
+        update-types: ['version-update:semver-patch']
+    reviewers:
+      - 'OffchainLabs/full-stack'
+    labels:
+      - 'dependencies'
+      - 'internal'
+
+  # External dependencies
+  - package-ecosystem: 'npm'
+    directory: '/src'
+    schedule:
+      interval: 'daily'
+    ignore:
+      - dependency-name: '@arbitrum/*'
+      - dependency-name: '@offchainlabs/*'
+    # Disable version updates for external dependencies - security only
+    open-pull-requests-limit: 0
+    reviewers:
+      - 'OffchainLabs/full-stack'
+    labels:
+      - 'dependencies'
+      - 'external'
```

Comment:
since we have disabled opening PRs for external deps, i don't think we need these?